django_cloud_deploy/workflow/__init__.py

```diff
@@ -417,16 +417,19 @@ def _generate_base_secrets(database_username: str,
 
     @staticmethod
     def _load_secret_names(required_service_accounts: Optional[
```

Comment:
I think that you  can remove the `Optional` - I looked at the call site and it can never be None.

Reply:
Then you can remove the two disables.

Reply:
Don!